pkg/apis/workflow/v1alpha1/cron_workflow_types.go

```diff
@@ -59,6 +59,15 @@ type CronWorkflowSpec struct {
 	Timezone string `json:"timezone,omitempty" protobuf:"bytes,8,opt,name=timezone"`
 	// WorkflowMetadata contains some metadata of the workflow to be run
 	WorkflowMetadata *metav1.ObjectMeta `json:"workflowMetadata,omitempty" protobuf:"bytes,9,opt,name=workflowMeta"`
+	// StopStrategy defines if the cron workflow will stop being triggered once a certain condition has been reached, involving a number of runs of the workflow
+	StopStrategy *StopStrategy `json:"stopStrategy,omitempty" protobuf:"bytes,10,opt,name=stopStrategy"`
+}
+
+// StopStrategy defines if the cron workflow will stop being triggered once a certain condition has been reached, involving a number of runs of the workflow
```

Comment:
```suggestion
// StopStrategy defines if the CronWorkflow should stop scheduling based on a condition
```
same as above, although I think the version specifier isn't necessary here since the Fields Reference doesn't pull this one

Reply:
I was wrong, this does get pulled into the Fields Reference and so does need a version specifier. Followed up in https://github.com/argoproj/argo-workflows/pull/12696#discussion_r1508529694